test/integration/features/basic.feature

```diff
@@ -34,9 +34,10 @@ Feature: Basic
     and OpenShift web console should be accessible.
 
     Given Minishift has state "Running"
-     When "status code" of HTTP request to "/healthz" of OpenShift instance is equal to "200"
-     Then "body" of HTTP request to "/healthz" of OpenShift instance contains "ok"
-      And with up to "20" retries with wait period of "500ms" the "status code" of HTTP request to "/console" of OpenShift instance is equal to "200"
+     Then with up to "20" retries with wait period of "3" seconds container name "k8s_webconsole_webconsole" should be "running"
+      And "status code" of HTTP request to "/healthz" of OpenShift instance is equal to "200"
+      And "body" of HTTP request to "/healthz" of OpenShift instance contains "ok"
+      And with up to "40" retries with wait period of "500ms" the "status code" of HTTP request to "/console" of OpenShift instance is equal to "200"
```

Comment:
@LalatenduMohanty Since now we are already validating the webconsole container do you still think now we increase the retry in this step from `20` to `40` ?